Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Control.cs

```diff
@@ -42,7 +52,27 @@ public float getValue()
             return value;
         }
 
-        public virtual void Start()
+        /// <summary>
+        /// Returns the current value mapped onto a range between 0% and 100%.
+        /// </summary>
+        /// <returns>Normalized Value</returns>
+        public float getNormalizedValue()
+        {
+            return Mathf.Abs(Mathf.Round((value - cvr.controlMin) / (cvr.controlMax - cvr.controlMin) * 100));
+        }
+
+        public void SetContent(GameObject content, bool hideContent)
```

Comment:
The more I think about it the more I think we should actually document the control. It would also make sense from a user perspective to make it easy for them to create new controls. The syntax is too much for me though right now so late in the evening.